src/components/search-form/components/packages-filter/packages-filter.js

```diff
@@ -42,22 +23,28 @@ const PackagesFilter = ({
   disabled,
   params,
   titlesFacets,
-  packagesFilterMap,
+  prevDataOfOptedPackage,
   results,
   onUpdate,
 }) => {
   const initialTitlesFacets = useRef(titlesFacets).current;
-  const { packageIds } = activeFilters;
+  const { packageIds: selectedPackageId = '' } = activeFilters;
 
   const dataOptions = useMemo(() => {
-    const options = titlesFacets.packages?.map(({ id, name, count }) => ({
+    if (selectedPackageId && !titlesFacets.packages) {
```

Comment:
I think in the requirements we need to show all package options even when one package was selected

Reply:
This block is for the missing option. When we have no results, we also have no options, and if there is a selected option, we show it as 0.

Reply:
ahh I get it